src/Compilers/CSharp/Test/Emit2/Semantics/PrimaryConstructorTests.cs

```diff
@@ -6065,6 +6065,40 @@ public void XmlDoc([CombinatorialValues("class ", "struct")] string keyword)
             Assert.Equal("", constructor.GetParameters()[0].GetDocumentationCommentXml());
         }
 
+        [Theory, CombinatorialData, WorkItem("https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1931501")]
+        public void XmlDoc_InsideType(
```

Comment:
>XmlDoc_InsideType [](http://example.com/codeflow?start=20&length=17)

It looks like the problem also affects records. Consider adding similar tests to record specific test files

Reply:
I think class and struct records are tested in different files